website/i18n/ru/docusaurus-plugin-content-docs/current/usage/methods.md

```diff
@@ -112,7 +112,78 @@ class SomeApiService::Posts::Create < SomeApiService::Base
 end
 ```
 
-## Сокращения для методов
+## Группа из нескольких методов
+
+Собрать в одну группу выполнение несколько методов можно при помощи метода `stage`.
+
+:::info
+
+Использование опции `position` для `make` будет сортировать только внутри `stage`.
+
+:::
+
+```ruby
+stage do
+  make :create_user!
+  make :create_blog_for_user!
+  make :create_post_for_user_blog!
+end
+```
+
+### Оборачивание
+
+Группу методов, находящийхся в `stage` можно обернуть во что-то.
+Например, это может быть `ActiveRecord::Base.transaction` от Rails.
+
+:::caution
+
+Вызов метода `wrap_in` должен находиться над вызовом метода `make`.
+
+:::
+
+```ruby
+stage do
+  # highlight-next-line
+  wrap_in ->(methods:) { ActiveRecord::Base.transaction { methods } }
+  
+  make :create_user!
+  make :create_blog_for_user!
+  make :create_post_for_user_blog!
+end
+```
+
+### Откат
+
+Если в одном из методов в группе или в `wrap_in` возникло исключение, то это можно обработать при помощи метода `rollback`.
+
+:::caution
+
+Вызов метода `rollback` должен находиться над вызовом метода `make`.
+
+:::
+
+```ruby
+stage do
+  wrap_in ->(methods:) { ActiveRecord::Base.transaction { methods } }
+  # highlight-next-line
+  rollback :clear_data_and_fail!
+  
+  make :create_user!
+  make :create_blog_for_user!
+  make :create_post_for_user_blog!
+end
+
+# ...
+
+# highlight-next-line
+def clear_data_and_fail!(e)
+  # ...
+
+  fail!(message: "Failed to create data: #{e.message}")
+end
+```
+
+## Сокращение для методов
 
 Через конфигурацию `method_shortcuts` можно добавить часто используемые слова, которые используются в виде префиксов в именах методов.
```

Comment:
Through the `method_shortcuts` configuration, you can add frequently used words that are used as prefixes in method names.